### DISCUSSION TITLE
Welcome to emgutf Discussions!

### DISCUSSION THREAD

**Discussion by emgucv** [emgucv/emgutf#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tkfc00** [comment]
Hi 
My name is franck. I'm a young student in medical technology , passionate about the bibliothek Emgu.CV precisely in calibration field. I have been using Emgu.Cv for atleast 7 month. It is very difficult because I have no one who can guide me when I'm facing dificulties . My university is not specialized in the field and I wish to specialized myself in this domain for professional reasons.
